src/style.css

```diff
@@ -1,81 +1,98 @@
 body {
-  box-shadow: 0 2px 2px 2px rgba(109, 97, 97, 0.3);
   font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
-  margin: 1rem 1rem 1rem 1rem;
+  border: 1px solid  rgba(54, 52, 52, 0.116);
+  margin: 2rem 2rem 2rem 2rem;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
 }
 
-#list_head {
+header {
+  border: 1px solid  rgba(54, 52, 52, 0.116);
   display: flex;
+  justify-content: space-between;
   align-items: center;
+  gap: 20px;
+  width: 100%;
+  height: 5rem;
 }
 
-#title {
-  width: 85%;
-  font-size: 1rem;
-  padding-top: 1rem;
-  margin: 0 1rem 1rem 1rem;
-  font-weight: bold;
+.title {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 2rem;
+  font-weight: 400;
+  font-size: 2rem;
 }
 
-.circlearrows {
-  content: "\21BB";
-  color: black;
-  font-weight: 700;
+.iconth {
+  margin-right: 2rem;
 }
 
-hr {
-  margin: 0;
-  padding: 0;
-  border: solid rgba(139, 121, 121, 0.137);
-  border-width: 1px;
+.input-add input {
+  margin-left: 2rem;
+  width: 100%;
+  border: none;
+  font-style: italic;
+  font-size: 1.5rem;
 }
 
-#addtask {
+.input-add {
+  border: 1px solid  rgba(54, 52, 52, 0.116);
+  height: 5rem;
   display: flex;
-  font-size: 1rem;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
-  margin: 0 1rem 0 1rem;
+  justify-content: start;
+  align-items: center;
```

Comment:
- [ ] [OPTIONAL] Your project is working as expected 🎉  but I think it would be much better to add a width to your add `input` field  and remove the outline

![image](https://user-images.githubusercontent.com/75126481/221069912-30fe42f2-cb06-4249-8dfd-b4ed96d3952c.png)